src/flutter/shell/platform/linux_embedded/surface/environment_egl.h

```diff
@@ -15,14 +18,31 @@ namespace flutter {
 template <typename T>
 class EnvironmentEgl {
  public:
-  EnvironmentEgl(T* platform_display)
+  EnvironmentEgl(T *platform_display)
```

Comment:
Why did you change?

Reply:
It is a change by Formatter of VScode. However, when I executed Formatter again after changing codes, it returned to the original state.